### PR TITLE
fix(do not block sell after trade)

### PR DESCRIPTION
**Pull request by sixtedemaupeou**
## Description
891

## Change Type
- Bug Fix

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## Code Checklist
- [x] Code compiles successfully (verified via `yarn start`)
- [x] No lint issues exist (verified via `yarn lint`)
- [x] New and existing unit tests pass (verified via `yarn test`)
- [x] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js [hunk 1]
```diff
@@ -148,12 +148,12 @@ const FiatConvertor = (props) => {
...
-            form === 'coinifyCheckoutBuy'
-              ? <FormattedMessage id='buy.quote_input.remaining_buy_limit' defaultMessage='Your remaining buy limit is {max}' values={{ max: <a onClick={() => setMax(limits.max)}>{curr}{limits.max}</a> }} />
-              : getSellLimits()
+            isSell
+              ? getSellLimits()
+              : <FormattedMessage id='buy.quote_input.remaining_buy_limit' defaultMessage='Your remaining buy limit is {max}' values={{ max: <a onClick={() => setMax(limits.max)}>{curr}{limits.max}</a> }} />
           }
           {
-            level.name < 2 && !equals(prop('state', kyc), 'reviewing')
+            gt(2, prop('name', level)) && not(equals(prop('state', kyc), 'reviewing'))
```
Comment (prwelber) [inline packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js]: Looks like the line taken out reads as "level less than 2 AND kyc does not equal reviewing"
The new line reads "level greater than 2 AND kyc does not equal reviewing"

Is the change intended here, and why? The only users who should see the increase limits message are level 1.

File: packages/blockchain-wallet-v4-frontend/src/services/CoinifyService/index.js [hunk 2]
```diff
@@ -249,16 +256,13 @@ export const kycNotificationButtonHelper = (status) => {
   }
 }
 
-export const showKycStatus = (state) => {
-  if (state === 'pending') return state
-  if (state === 'rejected') return state
-  return false
-}
+export const showKycStatus = (state) =>
+  any(equals(state), ['pending', 'rejected']) ? state : false
```
Comment (prwelber) [inline packages/blockchain-wallet-v4-frontend/src/services/CoinifyService/index.js]: nice refactor

**Comment by plondon** [comment]
I moved 891 back to in progress

File: packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js (hunk 1 above)
Comment (sixtedemaupeou) [reply packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js]: No, the new line does reads 2 greater than name of level and kyc not reviewing. See https://ramdajs.com/docs/#gt

Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js]: Ah, that's my bad. Will approve this then.

**Pull request merged**